apps/src/lab2/views/LabViewsRenderer.tsx

```diff
@@ -67,7 +82,14 @@ const appsProperties: {[appName in AppName]?: AppProperties} = {
   },
   pythonlab: {
     backgroundMode: false,
-    node: <PythonlabView />,
+    node: <div />,
+    lazyNode: lazy(() =>
+      import(
+        /* webpackChunkName: "pythonlab" */ '../../pythonlab/index.js'
```

Comment:
Out of curiosity - does this actually create a separate webpack chunk, or is this still technically part of the lab2.js chunk?

Reply:
My understanding is yes. This comment actually does some magic so we get an asset request for `pythonlab.js` over the network (without it the request is for a file with a number as a name like `215.js`).